solutions/4-getting-started-js/only.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable valid-typeof */
+// eslint-disable-next-line no-var
```

Comment:
You can allow using var in the global eslint config file instead of constantly disabling it.